src/pages/Home/Logo/Logo.tsx

```diff
@@ -0,0 +1,63 @@
+type PropsTitleBar = {
+	textcolor: string | undefined | null;
```

Comment:
small suggestion here:
```suggestion
	textcolor?: string;
```